css/custom.scss

```
@@ -330,4 +330,37 @@ $link-active: $link-hover;
 
 .peoples {
   margin-top: 1em;
-}
\ No newline at end of file
+}
+
+.button {
+   -webkit-border-radius: 20px;
+   -moz-border-radius: 20px;
+   border-radius: 20px;
+   height: 50px;
+   line-height: 50px;
+   color: #FFFFFF;
+   font-family: Arial;
+   width: 281px;
+   font-size: 20px;
+   font-weight: 900;
+   padding: 12px;
+   background-color: #48B87D;
```

```suggestion
   background-color: $dark-green;
```

use variable rather than hardcode